fem/fespace.hpp

```diff
@@ -150,27 +157,54 @@ class FiniteElementSpace
    mutable Array<FaceQuadratureInterpolator*> E2IFQ_array;
    mutable Array<FaceQuadratureInterpolator*> E2BFQ_array;
 
-   long sequence; // should match Mesh::GetSequence
+   long sequence; // to detect changes in the mesh: should match Mesh::GetSequence
+   bool orders_changed; // space needs updating (rebuilding) if true
+
+   bool relaxed_hp; // see SetRelaxedHpConformity()
```

Comment:
`relaxed_hp` is not initialized in some of the constructors of `FiniteElementSpace`, which caused me some undefined/nondeterministic behavior.

Reply:
Thanks, I'll fix that.